files/leaflet.esri.renderers/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Rachel Nehmer"
+github = "https://github.com/Esri/esri-leaflet-related.git"
```

Comment:
:D heh, I used to mess my submits all the time before I wrote a quick setup script https://github.com/megawac/cdn-it